### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by mphschmitt**
## Type of Change

- Add French translations

## What issue does this relate to?

- None

### What should this PR do?

- Add French translations

### CODE REVIEW

File: src/nginxconfig/i18n/fr/common.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    wordPress: 'WordPress',
+    drupal: 'Drupal',
+    magento: 'Magento',
+    joomla: 'Joomla',
+    django: 'Django',
+    logging: 'Logs',
+    reverseProxy: 'Reverse Proxy',
+    reverseProxyLower: 'reverse proxy',
+    restrict: 'Restreindre',
+    path: 'chemin',
```
Comment (MattIPv4) [inline src/nginxconfig/i18n/fr/common.js:47]:
```suggestion
    path: 'Chemin',
```
Uppercase first letter?

File: src/nginxconfig/i18n/fr/common.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    letsEncrypt: 'Let\'s Encrypt',
+    python: 'Python',
+    wordPress: 'WordPress',
+    drupal: 'Drupal',
+    magento: 'Magento',
+    joomla: 'Joomla',
+    django: 'Django',
+    logging: 'Logs',
+    reverseProxy: 'Reverse Proxy',
+    reverseProxyLower: 'reverse proxy',
```
Comment (MattIPv4) [inline src/nginxconfig/i18n/fr/common.js:45]: Can these be translated?

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
Two very minor things, otherwise lgtm, thank you very much for doing this!

**Comment by MattIPv4** [comment]
cc @moniang if you're lurking, `French` needs translating to Chinese 😛 💙

**Comment by moniang** [comment]
> cc @moniang如果您潜伏，`French`需要翻译成中文😛 💙

`法语`

**Comment by moniang** [comment]
> cc @moniang if you're lurking, `French` needs translating to Chinese 😛 💙

`法语`

File: src/nginxconfig/i18n/fr/common.js (hunk 1 above)
Comment (mphschmitt) [reply src/nginxconfig/i18n/fr/common.js]: Done.

File: src/nginxconfig/i18n/fr/common.js (hunk 2 above)
Comment (mphschmitt) [reply src/nginxconfig/i18n/fr/common.js]: Done.

**Review by MattIPv4 — APPROVED** [review]
Lgtm, thanks again! 💙

**Comment by MattIPv4** [comment]
Hey, @mphschmitt - Thanks a ton for this sweet PR! 😄

Would you please shoot me an email when you get a chance?
mcowley at digitalocean dot com 🎉

**Pull request merged**